ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -914,80 +858,138 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
-                    onChange={v => this.updateResearchPurpose('anticipatedFindings', v)}/>
+
+          {/*disseminate  research Section */}
+        <WorkspaceEditSection header={researchPurposeQuestions[5].header}
+                              description={researchPurposeQuestions[5].description} style={{width: '50rem'}} index='3.'>
+          <FlexRow>
+            <FlexColumn style={{flex: '1 1 0', marginLeft: '1rem'}}>
+              {disseminateFindings.slice(0, sliceByHalfLength(disseminateFindings) + 1).map(
+                (rp, i) => this.makeDisseminateForm(rp, i))}
+            </FlexColumn>
+            <FlexColumn style={{flex: '1 1 0', marginLeft: '1rem'}}>
+              {disseminateFindings.slice(sliceByHalfLength(disseminateFindings)).map(
+                (rp, i) => this.makeDisseminateForm(rp, i))}
+            </FlexColumn>
+          </FlexRow>
         </WorkspaceEditSection>
-        <WorkspaceEditSection required header={researchPurposeQuestions[4].header}>
-          <Link onClick={() => this.setState({showUnderservedPopulationDetails:
-              !this.state.showUnderservedPopulationDetails})}>
-            More info on underserved populations
-            {this.state.showUnderservedPopulationDetails ? <ClrIcon shape='caret' dir='up'/> :
-              <ClrIcon shape='caret' dir='down'/>}
-          </Link>
-          {this.state.showUnderservedPopulationDetails && <div style={styles.text}>
-            A primary mission of the <i>All of Us</i> Research Program is to include research
-            participants who are medically underserved or are historically underrepresented in
-            Biomedical Research, or who, because of systematic social disadvantage, experience
-            health disparities.  As a way to assess the research being conducted with a focus on
-            these populations, <i>All of Us</i> requires that you indicate the demographic
-            categories you intend to focus your analysis on.
-          </div>}
-          <div style={{marginTop: '0.5rem'}}>
-            <RadioButton name='population' style={{marginRight: '0.5rem'}}
-                         data-test-id='specific-population-no'
-                         onChange={v => this.updateResearchPurpose('population', false)}
-                         checked={!this.state.workspace.researchPurpose.population}/>
-            <label style={styles.text}>No, I am not interested in focusing on
-              specific population(s) in my research.</label>
-          </div>
+
+          {/*Research outcome section*/}
+          <WorkspaceEditSection header={researchPurposeQuestions[6].header} index='4.'
+                                description={researchPurposeQuestions[6].description}
+                                style={{width: '50rem'}}>
+            <FlexRow style={{marginLeft: '1rem'}}>
+              <FlexColumn style={{flex: '1 1 0'}}>
+                {researchOutcomings.map(
```

Comment:
[non-opt] The "outcomings" --> "outcomes" typo was fixed in the API, but not the UI code. Please fix here as well.